src/components/Results/Submission.vue

```diff
@@ -138,9 +138,11 @@ export default {
 			font-weight: bold;
 		}
 
-		&-menu {
-			display: inline-block;
-		}
+	}
+	
+	&-menu {
+		margin: 0 0 12px 12px;
```

Comment:
```suggestion
		margin: 0 0 12px var(--default-grid-baseline);
```
I think the left margin of 12px is too big. The `var(--default-grid-baseline)` fits much better in my opinion.